presidio-analyzer/presidio_analyzer/predefined_recognizers/aba_routing_recognizer.py

```diff
@@ -0,0 +1,60 @@
+from presidio_analyzer import Pattern, PatternRecognizer
+
+
+class AbaRoutingRecognizer(PatternRecognizer):
+    """
+    Recognizes American Banking Association (ABA) routing number.
+    Also known as routing transit number (RTN) and used to identify financial
+    institutions and process transactions.
+    """
+
+    PATTERNS = [
+        Pattern("ABA routing number (weak)", r"\b[0123678]\d{8}\b", 0.05,),
+        Pattern("ABA routing number",
+                r"\b[0123678]\d{3}-\d{4}-\d\b",
+                0.3,),
+    ]
+
+    CONTEXT = [
+        "aba",
+        "routing",
+        "abarouting",
+        "association",
+        "bankrouting",
```

Comment:
is it necessary to implement these context words if we have a checksum validation?
I noticed that whenever validation passes, the accuracy always becomes 100%

Reply:
It's not necessary, however it would be used anyway (as it uses the context words for every recognizer)